### DISCUSSION TITLE
Unexpected default sidebar behaviour

### DISCUSSION THREAD

**Discussion by timmyomahony** [requarks/wiki#3424] [Help / Questions]
The default sidebar behaviour seems counter-intuitive to me (maybe I'm misunderstanding this but I thought this would be useful feedback).

I have created a root-page for a corresponding folder:

![](https://files.timmyomahony.com/Screen-Shot-2020-08-05-11-27-57.83-d8Bz4h/Screen-Shot-2020-08-05-11-27-57.83.png)

When I open this folder, as expected I see this page as part of the navigation:

![](https://files.timmyomahony.com/Screenshot-2020-08-05-at-11.30.05-uT5u76/Screenshot-2020-08-05-at-11.30.05.png)

1. My intuition is that this root-page should load automatically when I enter the folder. At the moment it doesn't, you just see it at the top of the folder nativation (i.e. the arrow above) but you have to actually click it.
2. When I click the root-page it shows the page as expected, but the sidebar has now completely reset, so the folder I thought I was in ("project management") is now closed and I've lost my place

Here's an example GIF of what I mean on the Wiki.js docs:

![](https://files.timmyomahony.com/Screen-Recording-2020-08-05-11-25-35-rHs3uW/Screen-Recording-2020-08-05-11-25-35.gif)

This feels very counter intuitive. It might be the case I'm setting up my pages incorrectly but I feel like this is very confusing behaviour.

**Comment by thoni56** [comment]
You are one of very many that have this experience with the "tree navigation". It seems that the primary focus in development has been on the "Static Navigation", because there are some really stunning documentation sites built with Wiki.js out there.

But myself included, there are many that indicate that the navigation is not what we expect if you go for "Site Tree" or "Custom Navigation".

I was hoping that the "Site Tree" navigation would be more like the "IndexMenu" that exists for Dokuwiki... Maybe I'll make a stab at providing this.

But for now there is at least an effort to change the worst part of it in a [PR](https://github.com/Requarks/wiki/pull/2933) and a [feature request](https://requests.requarks.io/wiki/p/change-site-tree-navigation-to-redirect-to-folder-landing-pages), please support them.

I've also created a [feature request](https://requests.requarks.io/wiki/p/full-tree-navigation-like-indexmenu) for that full tree navigation.

**Comment by rrg92** [comment]
same perception here. I guess that @timmyomahony showed the right topics need to be enhanced to a better experience
